### DISCUSSION TITLE
Doccano failed to start webserver

### DISCUSSION THREAD

**Discussion by 33zeng** [doccano/doccano#1876] [Q&A]
the command prompt did not respond and did not report an error 
![image](https://user-images.githubusercontent.com/108190625/175782812-44f5d9c4-15d8-46a8-afc0-ca1855db281d.png)

**Comment by 33zeng** [comment]
please!

**Reply by david-engelmann** [reply]
Have you configured it correctly? I usually use the docker-compose approach because of it's easy of use
